SportsHub/SportsHub.Api/Program.cs

```diff
@@ -81,13 +81,16 @@
 builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(configuration.GetConnectionString(SportsHubConstants.DbConnectionString)));
 builder.Services.AddTransient<IUnitOfWork, UnitOfWork>();
 builder.Services.AddTransient<IUserService, UserService>();
+builder.Services.AddTransient<IArticleService, ArticleService>();
+builder.Services.AddTransient<ICommentService, CommentService>();
 builder.Services.AddTransient<IAuthenticationService, AuthenticationService>();
 builder.Services.AddTransient<IJsonTokenService, JsonTokenService>();
 builder.Services.AddTransient<ExceptionHandler>();
 
 //Adding AutoMapper
 //Looks in the assembly the file is located for mapping profiles.
 builder.Services.AddAutoMapper(typeof(UserMapping));
```

Comment:
There is an overload of this method which gets Assembly collection. Please take a look at it.

Reply:
From what I understand, I should have all service interfaces + `IUnitOfWork` implement the same base interface in order to do that, is this correct?

Reply:
I understood the question was about the services, but it is for AutoMapper. Implemented using this [article](https://dev.to/sumitkharche/how-to-integrate-automapper-in-asp-net-core-web-api-3cn4#:~:text=The%20AddAutoMapper%20method%20provided%20by%20the%20AutoMapper%20package,single%20assembly%20or%20list%20of%20assemblies%20as%20input.).